### DISCUSSION TITLE
Unable to change branch name for edit commits

### DISCUSSION THREAD

**Discussion by DrJacobHolden** [community/community#135496] [Mobile]
When editing through the GitHub mobile app I am only able to commit to DrJacobHolden-Patch-1. 

I can see this generated branch name but I am unable to edit it. 

If the branch already exists or has been deleted I am not able to save any changes as the process to recreate this duplicate branch fails. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.172.0 (138201153)
Device: iPad16,3 (iOS 17.5.1)
</details>

**Comment by taki-on** [comment]
@DrJacobHolden thank you for sharing the issue. We currently do not support editing branch name when creating a new branch while editing a file. I will create an issue to track this and discuss it with the design team. Thanks!

**Reply by skipishere** [reply]
Any progress on this? Seems like quite a big miss and something that should be easy to add, makes editing a file in the mobile app a bit unhelpful
